src/status_im2/contexts/quo_preview/tabs/tabs.cljs

```diff
@@ -13,10 +13,21 @@
                :value "32"}
               {:key   24
                :value "24"}]}
+   {:label "Show unread indicators?"
```

Comment:
There are a few string labels. Not sure if they are displayed in the UI. If so, may need translation.

Reply:
These are just for the preview area :)